x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_indices/indices_logic.ts

```diff
@@ -72,8 +72,10 @@ export interface IndicesValues {
   deleteModalIndex: ElasticsearchViewIndex | null;
   deleteModalIndexName: string;
   deleteModalIngestionMethod: IngestionMethod;
+  deleteStatus: typeof DeleteIndexApiLogic.values.status;
```

Comment:
nitpick: this should just be typed as Status, which is already imported in this file.

Reply:
@sphilipse on L84 we also have `status: typeof FetchIndicesAPILogic.values.status;`. Should that also be changed to `status: Status;`?

Reply:
Probably yes! Not blocking either way.